README.md

```diff
@@ -113,7 +113,7 @@ A record of chain configurations. The record key is the chain ID. For example:
 }
 ```
 
-##### `contracts` _(optional)_
+##### `contracts`
 
 A record of contract addresses used by Airseeker. If not specified, the addresses are loaded from
```

Comment:
Given the above this may no longer apply (it's no longer optional)?

Reply:
Yeah. Thanks for noticing.